main/adapters/repository/protocol.go

```diff
@@ -160,6 +160,36 @@ func (p *ExtendedProtocol) StoreSignature(tx TransactionCtx, uid uuid.UUID, sign
 	return tx.Commit()
 }
 
+func (p *ExtendedProtocol) StoreExternalIdentity(ctx context.Context, extId ent.ExternalIdentity) (err error) {
+	// store public key raw bytes
+	extId.PublicKey, err = p.PublicKeyPEMToBytes(extId.PublicKey)
+	if err != nil {
+		return err
+	}
+
+	return p.ContextManager.StoreExternalIdentity(ctx, extId)
+}
+
+func (p *ExtendedProtocol) LoadExternalIdentity(ctx context.Context, uid uuid.UUID) (*ent.ExternalIdentity, error) {
+	extId, err := p.ContextManager.LoadExternalIdentity(ctx, uid)
+	if err != nil {
+		return nil, err
+	}
+
+	extId.PublicKey, err = p.PublicKeyBytesToPEM(extId.PublicKey)
```

Comment:
Same comment as in https://github.com/ubirch/ubirch-client-go/pull/103/files#diff-93f99157710c53e7b41d24bcc094a0819e9ea9a093aa9a3c9450ebf09534500cR165 applies here.

Reply:
See resolution in linked comment.